.circleci/config.yml

```diff
@@ -1,2907 +1,70 @@
 version: 2.1
+setup: true
```

Comment:
Any tips to review this PR? There's a lot of red and green, but I suspect the code changes are quite minimal?

Reply:
workflows.yml should be 1:1 with develop, so copy & paste and see nothing changed? I was bummed the diff on config.yml was so rough

Reply:
Right, ok, so I just need to really look at https://github.com/cypress-io/cypress/blob/1ac9a10649f283841688fc54d665553052506c29/.circleci/config.yml#L26 for new code.

Bash is gnarly, if it works it works -- cool hack on downloading the workflows.yml: https://github.com/cypress-io/cypress/blob/1ac9a10649f283841688fc54d665553052506c29/.circleci/config.yml#L61

Reply:
I see little to no risk in this change, so :heavy_check_mark: